arroyo-storage/src/lib.rs

```diff
@@ -119,6 +119,28 @@ fn matchers() -> &'static HashMap<Backend, Vec<Regex>> {
     })
 }
 
+macro_rules! retry {
+    ($e:expr) => {{
+        use std::thread::sleep;
+        use std::time::Duration;
+        let mut retries = 0;
+        let max_retries: u32 = 5;
```

Comment:
If I've done my math right, this is about 600ms from first to last retry. That seems a bit low for a potentially short-lived transient issue. Maybe something closer to 10 seconds would be good?